### ISSUE TITLE
Please support --swift-version in the pod_push action

### ISSUE THREAD

**Issue by marcuswu** [fastlane/fastlane#10569]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description
Even with a .swift-version file set correctly, I cannot get a pod lint to succeed. I can get it to succeed if I manually run pod spec lint --swift-version=4.0. I would like the pod_push action to support passing --swift-version to pod spec lint.

##### Complete output when running fastlane, including the stack trace and command used
> You can use: `--capture_output` as the last commandline argument to get that collected for you

### Captured Output

Command Used: `deploy`
<details><summary>Output/Log</summary>

```

Successfully loaded Appfile at path '<redacted>/Appfile'
- app_identifier: '<redacted>'
- apple_id: '<redacted>'
- team_id: '<redacted>'
-------

Merge branch 'fix_project' into 'master'

fix project

See merge request !19


+------+------------------------+-------------+
|              fastlane summary               |
+------+------------------------+-------------+
| Step | Action                 | Time (in s) |
+------+------------------------+-------------+
| 1    | Verifying required     | 0           |
|      | fastlane version       |             |
| 2    | default_platform       | 0           |
| 3    | git pull origin        | 2           |
|      | master                 |             |
| 4    | ensure_git_branch      | 0           |
| 5    | changelog_from_git_co  | 0           |
|      | mmits                  |             |
| 6    | git fetch --tags       | 2           |
| 💥   | pod_push               | 2           |
+------+------------------------+-------------+


Looking for related GitHub issues on fastlane/fastlane...
Search query: Exit status of command 'pod repo push Specs './CommonUtils.podspec' --sources='git@gitlab.<redacted>:iOS-CocoaPods/Specs.git,https://github.com/CocoaPods/Specs.git' --allow-warnings --verbose' was 1 instead of 0.

Validating spec
 -> CommonUtils
  CommonUtils (0.1.19) - Analyzing on iOS 9.0 platform.
  Preparing

Analyzing dependencies

Inspecting targets to integrate
  Using `ARCHS` setting to build architectures of target `Pods-App`: (``)

Fetching external sources
-> Fetching podspec for `CommonUtils` from `<redacted>/iOS-CocoaPods/CommonUtils/CommonUtils.podspec`

Resolving dependencies of 

Comparing resolved specification to the sandbox manifest
  A CommonUtils
  A PhoneNumberKit

Downloading dependencies

-> Installing CommonUtils (0.1.19)
  > Copying CommonUtils from `<redacted>/Library/Caches/CocoaPods/Pods/External/CommonUtils/393fb4ad37fefe8e2e834c99d7c31f25-6bd1e` to `../../../../../private/var/folders/s7/ytzbq2dn0kgfr7tg696xsh1w0000gp/T/CocoaPods-Lint-20171011-57380-1944qc7-CommonUtils/Pods/CommonUtils`

-> Installing PhoneNumberKit (2.0.0)
  > Copying PhoneNumberKit from `<redacted>/Library/Caches/CocoaPods/Pods/Release/PhoneNumberKit/2.0.0-073b3` to `../../../../../private/var/folders/s7/ytzbq2dn0kgfr7tg696xsh1w0000gp/T/CocoaPods-Lint-20171011-57380-1944qc7-CommonUtils/Pods/PhoneNumberKit`
  - Running pre install hooks

Generating Pods project
  - Creating Pods project
  - Adding source files to Pods project
  - Adding frameworks to Pods project
  - Adding libraries to Pods project
  - Adding resources to Pods project
  - Linking headers
  - Installing targets
    - Installing target `CommonUtils` iOS 9.0
      - Generating Info.plist file at `../../../../../private/var/folders/s7/ytzbq2dn0kgfr7tg696xsh1w0000gp/T/CocoaPods-Lint-20171011-57380-1944qc7-CommonUtils/Pods/Target Support Files/CommonUtils/Info.plist`
      - Generating module map file at `../../../../../private/var/folders/s7/ytzbq2dn0kgfr7tg696xsh1w0000gp/T/CocoaPods-Lint-20171011-57380-1944qc7-CommonUtils/Pods/Target Support Files/CommonUtils/CommonUtils.modulemap`
      - Generating umbrella header at `../../../../../private/var/folders/s7/ytzbq2dn0kgfr7tg696xsh1w0000gp/T/CocoaPods-Lint-20171011-57380-1944qc7-CommonUtils/Pods/Target Support Files/CommonUtils/CommonUtils-umbrella.h`
    - Installing target `PhoneNumberKit` iOS 8.0
      - Generating Info.plist file at `../../../../../private/var/folders/s7/ytzbq2dn0kgfr7tg696xsh1w0000gp/T/CocoaPods-Lint-20171011-57380-1944qc7-CommonUtils/Pods/Target Support Files/PhoneNumberKit/Info.plist`
      - Generating module map file at `../../../../../private/var/folders/s7/ytzbq2dn0kgfr7tg696xsh1w0000gp/T/CocoaPods-Lint-20171011-57380-1944qc7-CommonUtils/Pods/Target Support Files/PhoneNumberKit/PhoneNumberKit.modulemap`
      - Generating umbrella header at `../../../../../private/var/folders/s7/ytzbq2dn0kgfr7tg696xsh1w0000gp/T/CocoaPods-Lint-20171011-57380-1944qc7-CommonUtils/Pods/Target Support Files/PhoneNumberKit/PhoneNumberKit-umbrella.h`
    - Installing target `Pods-App` iOS 9.0
      - Generating Info.plist file at `../../../../../private/var/folders/s7/ytzbq2dn0kgfr7tg696xsh1w0000gp/T/CocoaPods-Lint-20171011-57380-1944qc7-CommonUtils/Pods/Target Support Files/Pods-App/Info.plist`
      - Generating module map file at `../../../../../private/var/folders/s7/ytzbq2dn0kgfr7tg696xsh1w0000gp/T/CocoaPods-Lint-20171011-57380-1944qc7-CommonUtils/Pods/Target Support Files/Pods-App/Pods-App.modulemap`
      - Generating umbrella header at `../../../../../private/var/folders/s7/ytzbq2dn0kgfr7tg696xsh1w0000gp/T/CocoaPods-Lint-20171011-57380-1944qc7-CommonUtils/Pods/Target Support Files/Pods-App/Pods-App-umbrella.h`
  - Running post install hooks
  - Writing Xcode project file to `../../../../../private/var/folders/s7/ytzbq2dn0kgfr7tg696xsh1w0000gp/T/CocoaPods-Lint-20171011-57380-1944qc7-CommonUtils/Pods/Pods.xcodeproj`
  - Writing Lockfile in `../../../../../var/folders/s7/ytzbq2dn0kgfr7tg696xsh1w0000gp/T/CocoaPods-Lint-20171011-57380-1944qc7-CommonUtils/Podfile.lock`
  - Writing Manifest in `../../../../../private/var/folders/s7/ytzbq2dn0kgfr7tg696xsh1w0000gp/T/CocoaPods-Lint-20171011-57380-1944qc7-CommonUtils/Pods/Manifest.lock`

Integrating client project

[!] Please close any current Xcode sessions and use `App.xcworkspace` for this project from now on.

Integrating target `Pods-App` (`../../../../../var/folders/s7/ytzbq2dn0kgfr7tg696xsh1w0000gp/T/CocoaPods-Lint-20171011-57380-1944qc7-CommonUtils/App.xcodeproj` project)
  Adding Build Phase '[CP] Embed Pods Frameworks' to project.
  Adding Build Phase '[CP] Copy Pods Resources' to project.
  Adding Build Phase '[CP] Check Pods Manifest.lock' to project.
  - Running post install hooks

-> Pod installation complete! There is 1 dependency from the Podfile and 2 total pods installed.
  Building with xcodebuild.
xcodebuild clean build -workspace /var/folders/s7/ytzbq2dn0kgfr7tg696xsh1w0000gp/T/CocoaPods-Lint-20171011-57380-1944qc7-CommonUtils/App.xcworkspace -scheme App -configuration Release CODE_SIGN_IDENTITY=- -sdk iphonesimulator -destination id=B6ACD72B-A01C-4D86-8C63-B126BE0D8322
Build settings from command line:
    CODE_SIGN_IDENTITY = -
    SDKROOT = iphonesimulator11.0

=== CLEAN TARGET PhoneNumberKit OF PROJECT Pods WITH CONFIGURATION Release ===

Check dependencies
The “Swift Language Version” (SWIFT_VERSION) build setting must be set to a supported value for targets which use Swift. This setting can be set in the build settings editor.
The “Swift Language Version” (SWIFT_VERSION) build setting must be set to a supported value for targets which use Swift. This setting can be set in the build settings editor.

** CLEAN FAILED **


The following build commands failed:
	Check dependencies
(1 failure)
=== BUILD TARGET PhoneNumberKit OF PROJECT Pods WITH CONFIGURATION Release ===

Check dependencies
The “Swift Language Version” (SWIFT_VERSION) build setting must be set to a supported value for targets which use Swift. This setting can be set in the build settings editor.
The “Swift Language Version” (SWIFT_VERSION) build setting must be set to a supported value for targets which use Swift. This setting can be set in the build settings editor.

** BUILD FAILED **


The following build commands failed:
	Check dependencies
(1 failure)
  Testing with xcodebuild.
 -> CommonUtils (0.1.19)
    - WARN  | source: Git SSH URLs will NOT work for people behind firewalls configured to only allow HTTP, therefore HTTPS is preferred.
    - ERROR | [iOS] xcodebuild: Returned an unsuccessful exit code.

[!] The `./CommonUtils.podspec` specification does not validate.

/Library/Ruby/Gems/2.0.0/gems/cocoapods-1.4.0.beta.1/lib/cocoapods/command/repo/push.rb:142:in `block in validate_podspec_files'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-1.4.0.beta.1/lib/cocoapods/command/repo/push.rb:128:in `each'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-1.4.0.beta.1/lib/cocoapods/command/repo/push.rb:128:in `validate_podspec_files'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-1.4.0.beta.1/lib/cocoapods/command/repo/push.rb:73:in `run'
/Library/Ruby/Gems/2.0.0/gems/claide-1.0.2/lib/claide/command.rb:334:in `run'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-1.4.0.beta.1/lib/cocoapods/command.rb:52:in `run'
/Library/Ruby/Gems/2.0.0/gems/cocoapods-1.4.0.beta.1/bin/pod:55:in `<top (required)>'
/usr/local/bin/pod:23:in `load'
/usr/local/bin/pod:23:in `<main>'

URL: https://api.github.com/search/issues?q=Exit%20status%20of%20command%20'pod%20repo%20push%20Specs%20'./CommonUtils.podspec'%20--sources='git@gitlab.<redacted>:iOS-CocoaPods/Specs.git,https://github.com/CocoaPods/Specs.git'%20--allow-warnings%20--verbose'%20was%201%20instead%20of%200.%0A%0AValidating%20spec%0A%20-%3E%20CommonUtils%0D%20%20CommonUtils%20(0.1.19)%20-%20Analyzing%20on%20iOS%209.0%20platform.%0A%20%20Preparing%0A%0AAnalyzing%20dependencies%0A%0AInspecting%20targets%20to%20integrate%0A%20%20Using%20%60ARCHS%60%20setting%20to%20build%20architectures%20of%20target%20%60Pods-App%60:%20(%60%60)%0A%0AFetching%20external%20sources%0A-%3E%20Fetching%20podspec%20for%20%60CommonUtils%60%20from%20%60<redacted>/src/iOS-CocoaPods/CommonUtils/CommonUtils.podspec%60%0A%0AResolving%20dependencies%20of%20%0A%0AComparing%20resolved%20specification%20to%20the%20sandbox%20manifest%0A%20%20A%20CommonUtils%0A%20%20A%20PhoneNumberKit%0A%0ADownloading%20dependencies%0A%0A-%3E%20Installing%20CommonUtils%20(0.1.19)%0A%20%20%3E%20Copying%20CommonUtils%20from%20%60<redacted>/Library/Caches/CocoaPods/Pods/External/CommonUtils/393fb4ad37fefe8e2e834c99d7c31f25-6bd1e%60%20to%20%60../../../../../private/var/folders/s7/ytzbq2dn0kgfr7tg696xsh1w0000gp/T/CocoaPods-Lint-20171011-57380-1944qc7-CommonUtils/Pods/CommonUtils%60%0A%0A-%3E%20Installing%20PhoneNumberKit%20(2.0.0)%0A%20%20%3E%20Copying%20PhoneNumberKit%20from%20%60/<redacted>/Library/Caches/CocoaPods/Pods/Release/PhoneNumberKit/2.0.0-073b3%60%20to%20%60../../../../../private/var/folders/s7/ytzbq2dn0kgfr7tg696xsh1w0000gp/T/CocoaPods-Lint-20171011-57380-1944qc7-CommonUtils/Pods/PhoneNumberKit%60%0A%20%20-%20Running%20pre%20install%20hooks%0A%0AGenerating%20Pods%20project%0A%20%20-%20Creating%20Pods%20project%0A%20%20-%20Adding%20source%20files%20to%20Pods%20project%0A%20%20-%20Adding%20frameworks%20to%20Pods%20project%0A%20%20-%20Adding%20libraries%20to%20Pods%20project%0A%20%20-%20Adding%20resources%20to%20Pods%20project%0A%20%20-%20Linking%20headers%0A%20%20-%20Installing%20targets%0A%20%20%20%20-%20Installing%20target%20%60CommonUtils%60%20iOS%209.0%0A%20%20%20%20%20%20-%20Generating%20Info.plist%20file%20at%20%60../../../../../private/var/folders/s7/ytzbq2dn0kgfr7tg696xsh1w0000gp/T/CocoaPods-Lint-20171011-57380-1944qc7-CommonUtils/Pods/Target%20Support%20Files/CommonUtils/Info.plist%60%0A%20%20%20%20%20%20-%20Generating%20module%20map%20file%20at%20%60../../../../../private/var/folders/s7/ytzbq2dn0kgfr7tg696xsh1w0000gp/T/CocoaPods-Lint-20171011-57380-1944qc7-CommonUtils/Pods/Target%20Support%20Files/CommonUtils/CommonUtils.modulemap%60%0A%20%20%20%20%20%20-%20Generating%20umbrella%20header%20at%20%60../../../../../private/var/folders/s7/ytzbq2dn0kgfr7tg696xsh1w0000gp/T/CocoaPods-Lint-20171011-57380-1944qc7-CommonUtils/Pods/Target%20Support%20Files/CommonUtils/CommonUtils-umbrella.h%60%0A%20%20%20%20-%20Installing%20target%20%60PhoneNumberKit%60%20iOS%208.0%0A%20%20%20%20%20%20-%20Generating%20Info.plist%20file%20at%20%60../../../../../private/var/folders/s7/ytzbq2dn0kgfr7tg696xsh1w0000gp/T/CocoaPods-Lint-20171011-57380-1944qc7-CommonUtils/Pods/Target%20Support%20Files/PhoneNumberKit/Info.plist%60%0A%20%20%20%20%20%20-%20Generating%20module%20map%20file%20at%20%60../../../../../private/var/folders/s7/ytzbq2dn0kgfr7tg696xsh1w0000gp/T/CocoaPods-Lint-20171011-57380-1944qc7-CommonUtils/Pods/Target%20Support%20Files/PhoneNumberKit/PhoneNumberKit.modulemap%60%0A%20%20%20%20%20%20-%20Generating%20umbrella%20header%20at%20%60../../../../../private/var/folders/s7/ytzbq2dn0kgfr7tg696xsh1w0000gp/T/CocoaPods-Lint-20171011-57380-1944qc7-CommonUtils/Pods/Target%20Support%20Files/PhoneNumberKit/PhoneNumberKit-umbrella.h%60%0A%20%20%20%20-%20Installing%20target%20%60Pods-App%60%20iOS%209.0%0A%20%20%20%20%20%20-%20Generating%20Info.plist%20file%20at%20%60../../../../../private/var/folders/s7/ytzbq2dn0kgfr7tg696xsh1w0000gp/T/CocoaPods-Lint-20171011-57380-1944qc7-CommonUtils/Pods/Target%20Support%20Files/Pods-App/Info.plist%60%0A%20%20%20%20%20%20-%20Generating%20module%20map%20file%20at%20%60../../../../../private/var/folders/s7/ytzbq2dn0kgfr7tg696xsh1w0000gp/T/CocoaPods-Lint-20171011-57380-1944qc7-CommonUtils/Pods/Target%20Support%20Files/Pods-App/Pods-App.modulemap%60%0A%20%20%20%20%20%20-%20Generating%20umbrella%20header%20at%20%60../../../../../private/var/folders/s7/ytzbq2dn0kgfr7tg696xsh1w0000gp/T/CocoaPods-Lint-20171011-57380-1944qc7-CommonUtils/Pods/Target%20Support%20Files/Pods-App/Pods-App-umbrella.h%60%0A%20%20-%20Running%20post%20install%20hooks%0A%20%20-%20Writing%20Xcode%20project%20file%20to%20%60../../../../../private/var/folders/s7/ytzbq2dn0kgfr7tg696xsh1w0000gp/T/CocoaPods-Lint-20171011-57380-1944qc7-CommonUtils/Pods/Pods.xcodeproj%60%0A%20%20-%20Writing%20Lockfile%20in%20%60../../../../../var/folders/s7/ytzbq2dn0kgfr7tg696xsh1w0000gp/T/CocoaPods-Lint-20171011-57380-1944qc7-CommonUtils/Podfile.lock%60%0A%20%20-%20Writing%20Manifest%20in%20%60../../../../../private/var/folders/s7/ytzbq2dn0kgfr7tg696xsh1w0000gp/T/CocoaPods-Lint-20171011-57380-1944qc7-CommonUtils/Pods/Manifest.lock%60%0A%0AIntegrating%20client%20project%0A%0A[!]%20Please%20close%20any%20current%20Xcode%20sessions%20and%20use%20%60App.xcworkspace%60%20for%20this%20project%20from%20now%20on.%0A%0AIntegrating%20target%20%60Pods-App%60%20(%60../../../../../var/folders/s7/ytzbq2dn0kgfr7tg696xsh1w0000gp/T/CocoaPods-Lint-20171011-57380-1944qc7-CommonUtils/App.xcodeproj%60%20project)%0A%20%20Adding%20Build%20Phase%20'[CP]%20Embed%20Pods%20Frameworks'%20to%20project.%0A%20%20Adding%20Build%20Phase%20'[CP]%20Copy%20Pods%20Resources'%20to%20project.%0A%20%20Adding%20Build%20Phase%20'[CP]%20Check%20Pods%20Manifest.lock'%20to%20project.%0A%20%20-%20Running%20post%20install%20hooks%0A%0A-%3E%20Pod%20installation%20complete!%20There%20is%201%20dependency%20from%20the%20Podfile%20and%202%20total%20pods%20installed.%0A%20%20Building%20with%20xcodebuild.%0Axcodebuild%20clean%20build%20-workspace%20/var/folders/s7/ytzbq2dn0kgfr7tg696xsh1w0000gp/T/CocoaPods-Lint-20171011-57380-1944qc7-CommonUtils/App.xcworkspace%20-scheme%20App%20-configuration%20Release%20CODE_SIGN_IDENTITY=-%20-sdk%20iphonesimulator%20-destination%20id=B6ACD72B-A01C-4D86-8C63-B126BE0D8322%0A2017-10-11%2016:25:43.350%20xcodebuild[57404:39977188]%20[MT]%20PluginLoading:%20Required%20plug-in%20compatibility%20UUID%20DF11C142-1584-4A99-87AC-1925D5F5652A%20for%20plug-in%20at%20path%20'~/Library/Application%20Support/Developer/Shared/Xcode/Plug-ins/XVim.xcplugin'%20not%20present%20in%20DVTPlugInCompatibilityUUIDs%0A2017-10-11%2016:25:43.351%20xcodebuild[57404:39977188]%20[MT]%20PluginLoading:%20Required%20plug-in%20compatibility%20UUID%20DF11C142-1584-4A99-87AC-1925D5F5652A%20for%20plug-in%20at%20path%20'~/Library/Application%20Support/Developer/Shared/Xcode/Plug-ins/Alcatraz.xcplugin'%20not%20present%20in%20DVTPlugInCompatibilityUUIDs%0ABuild%20settings%20from%20command%20line:%0A%20%20%20%20CODE_SIGN_IDENTITY%20=%20-%0A%20%20%20%20SDKROOT%20=%20iphonesimulator11.0%0A%0A===%20CLEAN%20TARGET%20PhoneNumberKit%20OF%20PROJECT%20Pods%20WITH%20CONFIGURATION%20Release%20===%0A%0ACheck%20dependencies%0AThe%20%E2%80%9CSwift%20Language%20Version%E2%80%9D%20(SWIFT_VERSION)%20build%20setting%20must%20be%20set%20to%20a%20supported%20value%20for%20targets%20which%20use%20Swift.%20This%20setting%20can%20be%20set%20in%20the%20build%20settings%20editor.%0AThe%20%E2%80%9CSwift%20Language%20Version%E2%80%9D%20(SWIFT_VERSION)%20build%20setting%20must%20be%20set%20to%20a%20supported%20value%20for%20targets%20which%20use%20Swift.%20This%20setting%20can%20be%20set%20in%20the%20build%20settings%20editor.%0A%0A**%20CLEAN%20FAILED%20**%0A%0A%0AThe%20following%20build%20commands%20failed:%0A%09Check%20dependencies%0A(1%20failure)%0A===%20BUILD%20TARGET%20PhoneNumberKit%20OF%20PROJECT%20Pods%20WITH%20CONFIGURATION%20Release%20===%0A%0ACheck%20dependencies%0AThe%20%E2%80%9CSwift%20Language%20Version%E2%80%9D%20(SWIFT_VERSION)%20build%20setting%20must%20be%20set%20to%20a%20supported%20value%20for%20targets%20which%20use%20Swift.%20This%20setting%20can%20be%20set%20in%20the%20build%20settings%20editor.%0AThe%20%E2%80%9CSwift%20Language%20Version%E2%80%9D%20(SWIFT_VERSION)%20build%20setting%20must%20be%20set%20to%20a%20supported%20value%20for%20targets%20which%20use%20Swift.%20This%20setting%20can%20be%20set%20in%20the%20build%20settings%20editor.%0A%0A**%20BUILD%20FAILED%20**%0A%0A%0AThe%20following%20build%20commands%20failed:%0A%09Check%20dependencies%0A(1%20failure)%0A%20%20Testing%20with%20xcodebuild.%0A%20-%3E%20CommonUtils%20(0.1.19)%0A%20%20%20%20-%20WARN%20%20%7C%20source:%20Git%20SSH%20URLs%20will%20NOT%20work%20for%20people%20behind%20firewalls%20configured%20to%20only%20allow%20HTTP,%20therefore%20HTTPS%20is%20preferred.%0A%20%20%20%20-%20ERROR%20%7C%20[iOS]%20xcodebuild:%20Returned%20an%20unsuccessful%20exit%20code.%0A%0A[!]%20The%20%60./CommonUtils.podspec%60%20specification%20does%20not%20validate.%0A%0A/Library/Ruby/Gems/2.0.0/gems/cocoapods-1.4.0.beta.1/lib/cocoapods/command/repo/push.rb:142:in%20%60block%20in%20validate_podspec_files'%0A/Library/Ruby/Gems/2.0.0/gems/cocoapods-1.4.0.beta.1/lib/cocoapods/command/repo/push.rb:128:in%20%60each'%0A/Library/Ruby/Gems/2.0.0/gems/cocoapods-1.4.0.beta.1/lib/cocoapods/command/repo/push.rb:128:in%20%60validate_podspec_files'%0A/Library/Ruby/Gems/2.0.0/gems/cocoapods-1.4.0.beta.1/lib/cocoapods/command/repo/push.rb:73:in%20%60run'%0A/Library/Ruby/Gems/2.0.0/gems/claide-1.0.2/lib/claide/command.rb:334:in%20%60run'%0A/Library/Ruby/Gems/2.0.0/gems/cocoapods-1.4.0.beta.1/lib/cocoapods/command.rb:52:in%20%60run'%0A/Library/Ruby/Gems/2.0.0/gems/cocoapods-1.4.0.beta.1/bin/pod:55:in%20%60%3Ctop%20(required)%3E'%0A/usr/local/bin/pod:23:in%20%60load'%0A/usr/local/bin/pod:23:in%20%60%3Cmain%3E'%0A+repo:fastlane/fastlane


```

</details>

<details><summary>✅ fastlane environment ✅</summary>

### Stack

| Key                         | Value                                                          |
| --------------------------- | -------------------------------------------------------------- |
| OS                          | 10.12.6                                                        |
| Ruby                        | 2.0.0                                                          |
| Bundler?                    | false                                                          |
| Git                         | git version 2.5.0                                              |
| Installation Source         | /usr/local/bin/fastlane                                        |
| Host                        | Mac OS X 10.12.6 (16G29)                                       |
| Ruby Lib Dir                | /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib |
| OpenSSL Version             | OpenSSL 0.9.8zc 19 Mar 2015                                    |
| Is contained                | false                                                          |
| Is homebrew                 | false                                                          |
| Is installed via Fabric.app | false                                                          |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/                    |
| Xcode Version               | 9.0                                                            |

### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | en_US.UTF-8 | ✅ |
| LC_ALL   |             |   |
| LANGUAGE |             |   |

### fastlane files:

<details><summary>`./fastlane/Fastfile`</summary>

```ruby
# Customise this file, documentation can be found here:
# https://github.com/fastlane/fastlane/tree/master/fastlane/docs
# All available actions: https://docs.fastlane.tools/actions
# can also be listed using the `fastlane actions` command

# Change the syntax highlighting to Ruby
# All lines starting with a # are ignored when running `fastlane`

# If you want to automatically update fastlane if a new version is available:
# update_fastlane

# This is the minimum version number required.
# Update this, if you use features of a newer version
fastlane_version "2.19.0"

default_platform :ios

platform :ios do
  before_all do
    @pod_name = "CommonUtils"
  end

  desc "Runs all the tests"
  lane :test do
    cocoapods(podfile: "Example/")
    scan(workspace: "./Example/#{@pod_name}.xcworkspace", scheme: "#{@pod_name}-Example")
  end

  desc "CocoaPod merge request"
  lane :merge_request do
    branch = git_branch
    if "master" == branch
        raise 'Cannot create a merge request from master!'
    end

    sh("git fetch --tags")
    current_version = `git for-each-ref --sort=-taggerdate --format '%(tag)' --count 1 refs/tags`.strip
    # make sure our podspec is at the latest version before bumping
    version = version_bump_podspec(path: "./#{@pod_name}.podspec", version_number: current_version)
    version = version_bump_podspec(path: "./#{@pod_name}.podspec")
    puts "Bumped podspec to version: #{version}"
    cocoapods(podfile: "Example/")

    # commit podspec version bump
    puts "Creating version tag #{version}"
    git_commit(path: ["./#{@pod_name}.podspec", "./Example/Podfile.lock"], message: "Bumped to version #{version}")
    sh("git tag -a #{version} -m \"Tagged with Fastlane\" ")
    push_to_git_remote(remote_branch: branch, force: false, tags: true)

    # submit MR to gitlab
    puts "Creating merge request"
    project_id = find_project_id(api_url: "https://gitlab.<redacted>")
    mr_title = branch.tr("_", " ")
    merge_request_url = create_merge_request(project_id: project_id, title: "WIP: #{mr_title}", api_url: "https://gitlab.<redacted>", assignee: "ianc", remove_source_branch: true)
    puts "Merge request successfully created as WIP at #{merge_request_url}. Edit and remove WIP when ready."
  end

  # After Step One - the branch needs to be approved on Gitlab and merged to master
  # once merging is done, step 2 updates the Specs repository
  desc "Pod deploy"
  lane :deploy do
    sh("git pull origin master")
    ensure_git_branch
    changelog = changelog_from_git_commits
    sh("git fetch --tags")
    pod_push(path: "./#{@pod_name}.podspec", repo: "Specs", allow_warnings: true, verbose: true, sources: ["git@gitlab.<redacted>:iOS-CocoaPods/Specs.git", "https://github.com/CocoaPods/Specs.git"])
    #pod_push(path: "./#{@pod_name}.podspec", repo: "Specs", allow_warnings: true, swift_version: 4.0, verbose: true, sources: ["git@gitlab.<redacted>:iOS-CocoaPods/Specs.git", "https://github.com/CocoaPods/Specs.git"])
  end
  after_all do |lane|
    # This block is called, only if the executed lane was successful
  end

  error do |lane, exception|
  end
end

```
</details>

<details><summary>`./fastlane/Appfile`</summary>

```ruby
app_identifier "<redacted>" # The bundle identifier of your app
apple_id "<redacted>" # Your Apple email address

team_id "<redacted>" # Developer Portal Team ID

# you can even provide different app identifiers, Apple IDs and team names per lane:
# More information: https://github.com/fastlane/fastlane/blob/master/fastlane/docs/Appfile.md

```
</details>

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.61.0  | ✅ Up-To-Date  |

### Loaded fastlane plugins:

**No plugins Loaded**

<details><summary><b>Loaded gems</b></summary>

| Gem                       | Version      |
| ------------------------- | ------------ |
| slack-notifier            | 1.5.1        |
| claide                    | 1.0.2        |
| colored2                  | 3.1.2        |
| nanaimo                   | 0.2.3        |
| xcodeproj                 | 1.5.2        |
| plist                     | 3.3.0        |
| multipart-post            | 2.0.0        |
| word_wrap                 | 1.0.0        |
| public_suffix             | 2.0.5        |
| tty-screen                | 0.5.0        |
| babosa                    | 1.0.2        |
| colored                   | 1.2          |
| highline                  | 1.7.8        |
| commander-fastlane        | 4.4.5        |
| unf_ext                   | 0.0.7.4      |
| unf                       | 0.1.4        |
| domain_name               | 0.5.20170404 |
| http-cookie               | 1.0.3        |
| faraday-cookie_jar        | 0.0.6        |
| fastimage                 | 2.1.0        |
| gh_inspector              | 1.0.3        |
| mini_magick               | 4.5.1        |
| multi_xml                 | 0.6.0        |
| security                  | 0.1.3        |
| xcpretty-travis-formatter | 0.0.4        |
| dotenv                    | 2.2.1        |
| declarative               | 0.0.10       |
| declarative-option        | 0.1.0        |
| representable             | 3.0.4        |
| mime-types-data           | 3.2016.0521  |
| mime-types                | 3.1          |
| httpclient                | 2.8.3        |
| google-api-client         | 0.13.6       |
| json                      | 2.1.0        |
| excon                     | 0.59.0       |
| rubyzip                   | 1.2.1        |
| faraday                   | 0.13.1       |
| faraday_middleware        | 0.12.2       |
| bundler                   | 1.15.4       |
| terminal-table            | 1.8.0        |
| unicode-display_width     | 1.3.0        |
</details>

*generated on:* **2017-10-11**
</details>

**Comment by janpio** [comment]
PR is already merged. Thanks!
